### PR TITLE
Update DataSourcingActor to accept current requests for inverters

### PR DESCRIPTION
**Pull request by shsms**
In some component configurations, it might be necessary to use current
readings from inverters to calculate grid current.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -71,3 +71,4 @@ This version ships an experimental version of the **Power Manager**, adds prelim
 - Fix rendering of diagrams in the documentation.
 - The `__getitem__` magic of the `MovingWindow` is fixed to support the same functionality that the `window` method provides.
 - Fixes incorrect implementation of single element access in `__getitem__` magic of `MovingWindow`.
+- Fix incorrect grid current calculations in locations where the calculations depeneded on current measurements from an inverter.
```
Comment (daniel-zullo-frequenz) [inline RELEASE_NOTES.md]:
```suggestion
- Fix incorrect grid current calculations in locations where the calculations depended on current measurements from an inverter.
```

File: src/frequenz/sdk/microgrid/component/_component_data.py [hunk 2]
```diff
@@ -231,6 +231,12 @@ class InverterData(ComponentData):
             -ve current means supply into the grid.
     """
 
+    current_per_phase: tuple[float, float, float]
+    """AC current in Amperes (A) for phase/line 1,2 and 3 respectively.
```
Comment (daniel-zullo-frequenz) [inline src/frequenz/sdk/microgrid/component/_component_data.py]:
```suggestion
    """AC current in Amperes (A) for phase/line 1, 2 and 3 respectively.
```

**Review by daniel-zullo-frequenz — COMMENTED** [review]
Only a couple of typos, LGTM

**Comment by shsms** [comment]
Thanks Dani, both done.

**Review by daniel-zullo-frequenz — APPROVED** [review]

**Pull request merged**